### PR TITLE
adds integration test

### PR DESCRIPTION
**Pull request by aleph1ow**

### CODE REVIEW

File: spec/features/as_a_whole/cangaroo_spec.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'Cangaroo', type: :request do
+  class FakeJob < Cangaroo::Job
```
Comment (iloveitaly) [inline spec/features/as_a_whole/cangaroo_spec.rb]: @aleph1ow Great idea to add some integration tests!

We should telling the polling infrastructure as well! What do you think about adding a `FakePollJob < Cangaroo::PollJob` spec?

**Comment by mtylty** [comment]
@aleph1ow please rebase on master as I've added better rubocop rules via #37

File: spec/features/as_a_whole/cangaroo_spec.rb (hunk 1 above)
Comment (mtylty) [inline spec/features/as_a_whole/cangaroo_spec.rb]: @iloveitaly yes fake jobs should be used for specs.

**Comment by mtylty** [comment]
@aleph1ow also, could you please fix the hound checks I've marked with a big thumb? 👍 

Thanks!

**Comment by AlessioRocco** [comment]
I've updated this PR with new integration specs and also some fixes, one of these fix the issue in this PR https://github.com/nebulab/cangaroo/pull/38

**Pull request merged**